### PR TITLE
Move code to use chroot or nmsenter executors in order to have single interface for different objects

### PR DESCRIPTION
**Pull request by tsorya**

### CODE REVIEW

File: main/main.go [hunk 1]
```diff
@@ -110,15 +111,19 @@ func main() {
...
 		TimestampFormat: "2006-01-02 15:04:05",
 		FullTimestamp:   true,
 	})
-	op := ops.NewOps(logr, ops.NewExecutor(logr, true))
-	rpmOstreeClient := rpmostreeclient.NewClient("ibu-imager", op)
-
 	log := ctrl.Log.WithName("controllers").WithName("ImageBasedUpgrade")
+
+	executor := ops.NewChrootExecutor(newLogger, true, utils.Host)
+	rpmOstreeClient := rpmostreeclient.NewClient("ibu-imager", executor)
```
Comment (jc-rh) [inline main/main.go:125]:
```suggestion
	rpmOstreeClient := rpmostreeclient.NewClient("ibu-controller", executor)
```

Comment (sudomakeinstall2) [reply main/main.go:125]: This client.id is not used anywhere. It is just an extra variable.

File: controllers/utils/cmd.go [hunk 2]
```diff
@@ -32,41 +25,3 @@ const (
 	PrepSetupStateroot string = "prepSetupStateroot.sh"
 	PrepCleanup        string = "prepCleanup.sh"
 )
```
Comment (sudomakeinstall2) [inline controllers/utils/cmd.go:27]: we can remove this file and move these constants to `constants.go`

File: main/main.go (hunk 1 above)
Comment (jc-rh) [reply main/main.go:125]: @donpenney this client is assigned to the ibu reconciler struct

**Pull request merged**